fuzz/dict

```diff
@@ -0,0 +1,31 @@
+"__ENCODING__"
```

Comment:
Could we expand this dictionary to include at least one of every token? For the most part you can just copy the examples out of `config.yml`